code/modules/power/supermatter/supermatter.dm

```diff
@@ -539,6 +539,20 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	if(!removed || !removed.total_moles() || isspaceturf(local_turf)) //we're in space or there is no gas to process
 		if(takes_damage)
 			damage += max((power / 1000) * DAMAGE_INCREASE_MULTIPLIER, 0.1) // always does at least some damage
+		if(!isspaceturf(local_turf) && produces_gas && power) //There is no gas to process, but we are not in a space turf. Lets make them.
```

Comment:
could you go based off the immutability of the gas mixture, rather then the turf being a space tile?

Reply:
Yes.